### DISCUSSION TITLE
unikraft-releases.nlehd.de not responding

### DISCUSSION THREAD

**Discussion by jonrob** [unikraft/unikraft#254] [Q&A] [answered]
I'm just trying to `kraft run` the python3 app to get a feel for Unikraft, but have found that the command hangs. Strace output makes it look like it's hanging on connect to `unikraft-releases.nlehd.de` / `195.37.154.13`. Is there a problem with the infrastructure? Is there a way to bypass fetching from here? 

Thanks, looks an interesting technology.

**Comment by nderjung** [comment, marked as answer]
Hey @jonrob,

We've experienced the same problem our end on the infrastructure side.  We're in the process of migrating the service, which should happen in the next two weeks.  For now, to avoid this problem, update your `.kraftrc` file like so:

```diff
  [fetch]
- prioritize_origin = false
+ prioritize_origin = true
```

**Reply by jonrob** [reply]
Very helpful, thanks. That's improved it.
